packages/client/src/components/measurements/MeasurementForm.re

```diff
@@ -16,7 +16,7 @@ let make = (~measurable: Types.measurable, ~defaultValueText="") => {
               measurable.id,
               value,
               competitorType,
-              description,
+              MeasurementCdfInput.MarkdownPreprocessing.addNewLines(description),
```

Comment:
If we do this, we should probably have it apply to all markdown renders, not just measurement inputs.

Reply:
> I think you've convinced me that this is a justified feature, but I think doing it well and right will be more work.

Plausibly. Another possibility would be to move to Github flavored markdown, or to some implementation (https://simplemde.com/?) which doesn't have the line break problem.

Reply:
Agree that it makes sense to have it on other renderers too

Reply:
I think Github does do this for new lines. I would prefer to move to Github flavored or another standard one that has them, instead of us coming up with our own hybrid, but was assuming that would be more work.

Reply:
So coming up with our hybrid might be a nightmare. For example, if you want to preserve newlines within codeblocks (\`\`\`), then you either have to write a parser, or some really ugly regular expressions. I was briefly looking into the Github Flavored Markdown specs: https://github.github.com/gfm, and I'm just glad someone else did it.

In this case I'd push for either.
1. Just do the hack for comments, but maybe not for notebooks
2. Move to github flavored markdown, or to another specification and/or a better editor.

I have some intuitions that in this case, the hack is better, but I also wouldn't mind the second option. I dislike the option in between, the "write your own principled hybrid".

Reply:
Apparently we may be able to implement github flavored markedown with our current react-markdown package, though of course I'm not sure exactly how hard it would be. https://github.com/rexxars/react-markdown/blob/c63dccb8185869cfc73c257d098a123ef7a7cd33/demo/src/demo.js

I'm hesitant to go forward with this as in (1), it just makes me feel uneasy. Hacks like that can really bite, we've probably gone too far that type of direction already.

Reply:
It's possible we can use a plugin for this. https://github.com/rexxars/react-markdown/issues/105

Reply:
Oh, cool